mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -95,7 +95,7 @@ public void start() {
                     // If remote URL is configured
                     if (mOpenHABUrl.length() > 0) {
                         Log.d(TAG, "Connecting to remote URL " + mOpenHABUrl);
-                        openHABTracked(mOpenHABUrl, mCtx.getString(R.string.info_conn_rem_url));
```

Comment:
ok, so why do we not want to give feedback to the user that the app is trying to connect to a URL?   I have seen where the app tries the local URL (which is not available) then tries the remote URL,  I think it's good that we give some feedback especially b/c in this case it adds a delay to loading a sitemap. Or is this something else?